src/app/video-courses/course-list/course-list.component.spec.ts

```diff
@@ -14,21 +15,21 @@ describe('CourseListComponent', () => {
       id: 123,
       title: 'Test Course 1',
       description: 'Description 1',
-      date: new Date(),
+      date: (new Date()).valueOf() - 500000000,
```

Comment:
Use `(new Date()).getTime()` as more convinient way to get timestamp in JS. `valueOf` is polymorphic method existing on a variety of Objects in JS, that used in comparison algorithm.

Reply:
Done